deployment/safe_haven_management_environment/cloud_init/resources/nexus_docker_compose.mustache.yaml

```
@@ -13,7 +13,7 @@ services:
     restart: always
   nginx:
     container_name: nginx_nexus_compose
-    image: nginx:1.21.6
+    image: nginx:1.25.3
```

```suggestion
    image: nginx:1.25
```

Might be better to avoid patch level here so that we get bug fixes.

---

Ok - same could apply to the other places we use nginx, so will go back and change that too

---

Yes, I think a good rule of thumb is pin on minor versions. That way you should avoid breaking changes, but get bug fixes.

Not all contains create a tag `x.y` which points to the latest minor release of `x` though.